mmengine/hooks/checkpoint_hook.py

```diff
@@ -128,6 +130,7 @@ def __init__(self,
                  file_client_args: Optional[dict] = None,
                  filename_tmpl: Optional[str] = None,
                  backend_args: Optional[dict] = None,
+                 save_start: int = 0,
```

Comment:
suggest renaming `save_start` to `begin`.

Reply:
emm, I think `save_start` more in line with what you see is what you get, more in line with what it means to start saving weight?

Reply:
Will `begin` be ambiguous? The function of this hook is to save checkpoints so `begin` should mean starting saving checkpoints.

Reply:
Okay. Can we rename it to `save_begin` which is consistent with `val_begin` at `mmengine/runner/loops.py`.

https://github.com/open-mmlab/mmengine/blob/4e685931d0687b45323b9811501490bfde2a6e11/mmengine/runner/loops.py#L26.

Reply:
ok，i agree with you